packages/x-charts/src/internals/plugins/models/seriesConfig/tooltipGetter.types.ts

```diff
@@ -4,9 +4,18 @@ import type {
   ChartSeriesDefaultized,
   ChartSeriesType,
 } from '../../../../models/seriesType/config';
+import { SeriesId } from '../../../../models/seriesType/common';
+import { AxisId } from '../../../../models/axis';
 
-export type TooltipGetter<T extends ChartSeriesType> = (params: {
-  series: ChartSeriesDefaultized<T>;
+export type TooltipGetter<TSeriesType extends ChartSeriesType> = (params: {
+  series: ChartSeriesDefaultized<TSeriesType>;
   getColor: (dataIndex: number) => string;
-  identifier: ChartItemIdentifier<T> | null;
-}) => ItemTooltip<T> | null;
+  identifier: ChartItemIdentifier<TSeriesType> | null;
+}) => ItemTooltip<TSeriesType> | null;
+
+export type AxisTriggeringTooltipGetter<
```

Comment:
Name is a bit odd, couldn't we have:

```suggestion
export type AxisTooltipGetter<
```

or

```suggestion
export type TooltipGetterAxes<
```

or

```suggestion
export type TooltipTriggerAxes<
```